README.md

```diff
@@ -1,11 +1,21 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="https://github.com/keploy/keploy/tree/main/src/images/logo.png" width="30%" />
```

Comment:
It would be awesome to use the image from Keploy org profile page.

Reply:
Done